flint.ui/src/views/flint/configurations/rothc/RothcSoilcover.vue

```diff
@@ -61,30 +76,54 @@
 
 <script>
 
-// import index from "src/"
-// import "@/store/index.js"
-// import { ref, reactive } from "vue";
-
 export default {
-
   components: {
     
   },
   computed: {
     newconfig_soilcover_data_orig: {
       get() {
-         return this.$store.state.Variables[8].rainfall.transform.data_orig
+         return this.$store.state.RothC_config.Variables[12].soilCover.transform.data_orig
       },
       set(newValue) {
-        // console.log(newValue)
         let tempval = newValue;
-        this.$store.commit('setNewConfig', tempval)
+        this.$store.commit('setNewConfig_soilCover_data_orig', tempval)
       }
-    }
+    },
+    
+    newconfig_soilcover_data_firstmonthcopy: {
```

Comment:
@waridrox - have you been able to verify whether any of these variables, other than `*_data_month_avg` are necessary to run?

If not, please file an issue and request they be removed to check whether the endpoint still works.